### DISCUSSION TITLE
Not working this project you wasted my whole time

### DISCUSSION THREAD

**Discussion by Namanj413** [Sudhanva-Nadiger/My-App#1] [Q&A] [answered]
Not working this project you wasted my whole time

![image](https://user-images.githubusercontent.com/103898041/227781817-73cc2320-e0de-4561-8c71-2ca5cee3517e.png)

**Comment by Sudhanva-Nadiger** [comment, marked as answer]
Hi !
It works fine in my system, !

![image](https://user-images.githubusercontent.com/93595710/227783118-090e6bdc-6ab6-4fde-87b5-f93e01b73bff.png)
![image](https://user-images.githubusercontent.com/93595710/227783193-9f015b8a-8d43-4c47-917f-d22cc4d2a095.png)

How can I host it without getting it run properly 😄?

- Possible fixes:
   -  make sure you have run npm install before starting the project!
   - If you have done that already try removing package lock file and then run npm install
   - add the extension to the path
    ```
    import bucketSlice from '../features/bucketSlice.js';
    import toggleSlice from "../features/toggleSlice.js";
    import historySlice from '../features/historySlice.js';
    ```

It may fix your problem! atleast thats what i can get from the error message you have given
